spec/support/synchronization_helpers.rb

```diff
@@ -1,7 +1,7 @@
 require 'English'
 
 module SynchronizationHelpers
-  def expect_in_fork(fork_expectations: nil)
+  def expect_in_fork(fork_expectations: nil, timeout_in_seconds: 10)
```

Comment:
Minor: I suggest using `timeout_seconds`, as there's a few places in the codebase that already use this pattern (e.g. without the in).